app/app.go

```diff
@@ -19,7 +19,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/baseapp"
```

Comment:
should x/group (https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.0) be added in this PR or a separate PR?

Reply:
maybe a different PR with corresponding integration test together?

Reply:
yes + upgrade handler